clients/unshaded/src/main/java/tachyon/client/OutStreamBase.java

```diff
@@ -26,7 +28,7 @@
  * the number of bytes written to the output stream, and extends {@link Cancelable} to abort the
  * writes.
  */
-@PublicApi
+@PublicApi @ThreadSafe
```

Comment:
Extensions of this abstract class are not thread safe, so this should `@NotThreadSafe`